src/cleanfigure.m

```diff
@@ -96,7 +96,8 @@ function cleanfigure(varargin)
 
   % Update the current axes.
   if strcmp(get(h, 'Type'), 'axes')
-      meta.gca = h;
+      meta.gca  = h;
+      meta.is3D = isAxis3D(h); 
```

Comment:
Is there a significant overhead for repeatedly calling the `isAxis3D` function? If not, I would opt to NOT cache its output as you do here.